is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Add a comment to describe what type of uniqueness you are looking for (unique characters? unique array items?)